src/main/java/frc/robot/subsystems/helicopter/Helicopter.java

```diff
@@ -164,37 +165,44 @@ public void setVelocity(double velocity) {
         }
     }
 
+    public void vroomVroom(double power) {
+        mainMotor.set(ControlMode.PercentOutput, power);
+    }
+
     /**
-     * Set climber position to zero.
+     * Set Helicopter position to zero.
      * Zero is the balanced position of the arms.
      */
     public void setAngleZero() {
         double angle = getAbsolutePosition();
-        setPosition(getPosition() - angle);
+        setPosition(getPosition().minus(new Rotation2d(angle)));
     }
 
     /**
-     * @return the absolute position of the Climber.
+     * @return the absolute position of the Helicopter.
      */
     public double getAbsolutePosition() {
-        return unitModelPosition.toUnits(dutyCycleEncoder.getDistance() - Constants.Climber.ZERO_POSITION);
+        return unitModelPosition.toUnits(dutyCycleEncoder.getDistance() - Constants.Helicopter.ZERO_POSITION);
     }
 
 
     /**
      * @return get motors position. [rad]
      */
-    public double getPosition() {
-        return unitModelPosition.toUnits(mainMotor.getSelectedSensorPosition(0));
+    public Rotation2d getPosition() {
+        return new Rotation2d(Math.IEEEremainder(unitModelPosition.toUnits(mainMotor.getSelectedSensorPosition(0)), Math.PI*2));
     }
 
     /**
      * @param position the position of the motors. [rad]
      */
-    public void setPosition(double position) {
-        mainMotor.set(ControlMode.MotionMagic, unitModelPosition.toTicks(position),
-                DemandType.ArbitraryFeedForward, feedforward.calculate(getPosition(), getVelocity()));
-    }
+    public void setPosition(Rotation2d position) {
+        var currentPosition = getPosition();
+        var error = position.minus(currentPosition);
+        Rotation2d minMove = new Rotation2d(Math.IEEEremainder(unitModelPosition.toTicks(error), Math.PI *2));
```

Comment:
You can just check whether the error is greater than Pi and if so reduce PI (and the opposite to -Pi)